ci-operator/step-registry/baremetal/lab/agent/install/baremetal-lab-agent-install-commands.sh

```diff
@@ -122,6 +129,7 @@ fi
 
 if [ "${masters}" -gt 1 ]; then
   if [ "${AGENT_PLATFORM_TYPE}" = "none" ]; then
+  yq --inplace 'del(.platform)' "${SHARED_DIR}"/install-config.yaml
```

Comment:
if this is because of the platform.baremetal.vips section, can we consider doing a definitive fix for this here? Maybe eliminating the AGENT_PLATFORM_TYPE and using two steps as you've done already for ipv6|4+fips conf should be enough.

Reply:
It has been handled within ipv6|4+fips conf file.